src/coreclr/vm/corelib.h

```diff
@@ -282,6 +282,9 @@ DEFINE_FIELD(DELEGATE,            METHOD_PTR_AUX,         _methodPtrAux)
 DEFINE_METHOD(DELEGATE,             CONSTRUCT_DELEGATE,     DelegateConstruct,          IM_Obj_IntPtr_RetVoid)
 DEFINE_METHOD(DELEGATE,             GET_INVOKE_METHOD,      GetInvokeMethod,            IM_RetIntPtr)
 
+DEFINE_CLASS(INT128,               System,                 Int128)
```

Comment:
Is this used anywhere?

Reply:
This is what defines `g_Int128` which is used in `methodTableBuilder.cpp` (https://github.com/dotnet/runtime/pull/69204/files/cb5a82e5df48bf4a7c6deea85765c75d3d4a2e03#diff-4b65f34bf68fdb80cc734cc2435c72ef7f86ef16b9f56af47c06b87761383d3aR9909)

Reply:
> This is what defines g_Int128

`g_Int128Name` is defined in classnames.h. This macro defines `CLASS_INT128` that I do not see used anywhere.

Reply:
Ah right, I was getting two two things mixed up.

Looks like this is missing handling in `classlayoutinfo.cpp` which copies the alignment requirement to the `pNativeLayoutInfo`: https://github.com/dotnet/runtime/blob/main/src/coreclr/vm/classlayoutinfo.cpp#L947-L964

Reply:
Should be fixed. The only other usage of `CLASS__*` for the `Vector*` types was blocking interop which isn't required for Int128/UInt128 given the interop tests are all passing.